app/views/sessions/dev_login.haml

```diff
@@ -0,0 +1,12 @@
+%h1 Development login
+
+
+= form_tag '/sessions/dev_login' do
+  = submit_tag 'Log in with no user record (should fail)'
+- [@admins, @dispatchers].each_with_index do |group, index|
+  %h2 Login as #{%w(Admin Dispatcher)[index]}:
```

Comment:
Rubocop would tell you that `%w` literals should be delimited with square brackets.